README.md

```diff
@@ -2,18 +2,17 @@
 
 # Project Name
 
-> Description the project.
+> Portfolio: setup and mobile version skeleton.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- CSS
+- HTML
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
+No live demo link
```

Comment:
- [ ] Since no demo link is available, it would be better if you remove this section completely from your README file 👍 
- [ ] Also I noticed some other empty sections in your README file, it would also be better to either fill in these sections or simply remove them in order to make your README file more professional.
- Your README
![Capture2](https://user-images.githubusercontent.com/85113567/153581603-cff13606-8757-43cd-9a88-8991a9c91cd1.PNG)